bundles/python/python_mode.moon

```diff
@@ -0,0 +1,34 @@
+-- Copyright 2012-2015 The Howl Developers
+-- License: MIT (see LICENSE.md at the top-level directory of the distribution)
+
+{
+  lexer: bundle_load('python_lexer')
+
+  comment_syntax: '#'
+
+  default_config:
+    word_pattern: r'\\b[\\w_][\\w\\d_]+\\b'
```

Comment:
This is not really sufficient for Python 3 I believe, since unicode is supported in variable and function names. This could I believe be supported by using the `\pL` unicode character class (used currently in places where unicode letters should be matched). See [this](https://developer.gnome.org/glib/stable/glib-regex-syntax.html#id-1.5.25.4.11) for reference. Apparently the same holds true for the Ruby mode it seems.